python/paddle/__init__.py

```diff
@@ -1173,4 +1174,5 @@
     'diagonal_scatter',
     'combinations',
     'signbit',
+    'set_',
```

Comment:
Do we need to support the usage of `paddle.set(x, source, ....)`? If it is not recommended for users to use it this way(recommended only `x.set(source, ...)` when x is `Tensor`), there is no need to add it here.

Reply:
This time only support `Tensor.set_` , this line can be deleted.

Reply:
deleted